zerver/openapi/openapi.py

```diff
@@ -117,28 +188,51 @@ def get_openapi_return_values(endpoint: str, method: str,
     response = response['properties']
     return response
 
-exclusion_list: List[str] = []
-
 def validate_against_openapi_schema(content: Dict[str, Any], endpoint: str,
                                     method: str, response: str) -> None:
     """Compare a "content" dict with the defined schema for a specific method
     in an endpoint.
     """
+    if endpoint in EXCLUDE_ENDPOINTS:
+        return
+    # Collision between /users/me/subscriptions/{id} and /users/{id}/subscripitions/{id}
+    if endpoint.startswith('/users/me/subscriptions/'):
+        return
+    # No 500 responses have been documented, so skip them
+    if response.startswith('5'):
+        return
+    if endpoint not in openapi_spec.spec()['paths'].keys():
+        match: bool = False
+        for keys in openapi_spec.regex_keys():
+            matches = re.match(fr'{keys}', endpoint)
+            if matches:
+                match = True
+                endpoint = openapi_spec.regex_keys()[keys]
+                break
+        # If it doesn't match it hasn't been documented yet.
```

Comment:
Did you

Reply:
Yes I ran zerver.tests with the print statement on. All of them are undocumented endpoints.